packages/react/src/hooks/useVoiceAssistant.ts

```diff
@@ -66,6 +78,7 @@ export function useVoiceAssistant(): VoiceAssistant {
     agent,
     state,
     audioTrack,
+    videoTrack,
```

Comment:
wouldn't we simply want to return all tracks that the agent publishes? 
not a fan of `audioTrack` in the first place, but adding `videoTrack` doesn't make it any better. 
Generally we have `microphoneTrack` and `cameraTrack` in our glossary of constructs, but not sure how well that works for agents

Reply:
yeah i don't have a strong opinion really. this seemed least disruptive and i think if you really want all the tracks you can get them from `agent` anyways